pkg/transaction/eventgenerator.go

```diff
@@ -170,6 +178,34 @@ func (e *Generator) createSamplingTransactionDetails(summaryEvent LogEvent) samp
 	}
 }
 
+// Validate APIs in the traceability exceptions list
+func (e *Generator) isInAPIExceptionsList(logEvents []LogEvent) bool {
+
+	// Check first leg for URI.  Use the raw value before redaction happens
+	uriRaw := ""
+
+	if httpEvent, ok := logEvents[0].TransactionEvent.Protocol.(*Protocol); ok {
+		uriRaw = httpEvent.URIRaw
+	}
+
+	// Get the api exceptions list
+	apiExceptionsList := traceability.GetAPIExceptionsList()
+	exceptions := strings.Split(apiExceptionsList, ",")
+	for i := range exceptions {
+		exceptions[i] = strings.TrimSpace(exceptions[i])
+	}
+
+	// If the api path exists in the exceptions list, return true and ignore event
+	for _, value := range exceptions {
+		if value == uriRaw {
+			log.Debugf("Found api path %s in traceability api exceptions list.  Ignore transaction event", uriRaw)
```

Comment:
Maybe the caller of this func should do the logging. The func returns T/F based on whether or not something is in the list. It could be called somewhere else for a completely different purpose, so this func shouldn't be in charge of deciding and logging that a transaction event is going to be ignored.

Reply:
i don't understand what you're saying?  what other purpose could it be called for, except checking if the path is in the list?

Reply:
The function is answering the question "is it in the list, yes or no". The first part of the debug is OK. But the second part about ignoring the event is a decision the caller of the function is making, so IMO that is where the "I'm gonna ignore this" should be. I know it's a little picky, but what if someone called the function only for the purpose of checking to see if something is in the list (which is what the function name implies). If it's in the list they want to do something, and if not something else. But in neither case are they going to "ignore transaction event". They will still see that in the logs.

Reply:
i lose the path to display since i'm only returning bool...  i guess i can return both?  i'll figure it out.